go.mod

```diff
@@ -102,6 +102,7 @@ require (
 	github.com/jackc/pgproto3/v2 v2.3.2
 	github.com/jackc/pgtype v1.14.0
 	github.com/jackc/pgx/v4 v4.18.1
+	github.com/jackc/pgx/v5 v5.4.2
```

Comment:
Do we need both packages? Shouldn't we stick to v4 for now?

Reply:
Why?

Reply:
Because I'm pretty sure that no one tests if v4 and v5 are compatible, and using both may have some unexpected side effects?

Reply:
They don't have to be compatible, they're completely independent implementations - that's the whole point of go module majors, you get to make breaking changes because the two versions can be used at the same time.

The only potential source of weirdness is if both `jackc/pgx/v4/stdlib` and `jackc/pgx/v5/stdlib` are imported, as that would make it unclear ([until Go 1.21](https://tip.golang.org/doc/go1.21#language)) which of the two `database/sql` drivers gets registered as `"pgx“` (they both also register `"pgx/v4"` and `"pgx/v5"` respectively). We use neither in the code, the only use of `github.com/jackc/pgx/v4/stdlib` is in a code example in the docs.

`pgx/v5` has a couple of new features that I'm relying on, `pgx.NamedArgs` being the harder to replace in a confidently safe way, and improved Batch ergonomics being another (not used here but in pgbk). I'd very much prefer to not have to update the code to go back to a previous major version.